database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,6 +17,7 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->string('password');
+            $table->integer('type');
```

Comment:
Another thing:- Since it is of type integer we need to agree on the value we are going to use for each role.
How about: 
Admin-> 00
Support Supervisor -> 01
Support Agent -> 10 ? 
Do you agree?